readme.MD

```diff
@@ -1,57 +1,106 @@
-
-## Authors
-
-:bust_in_silhouette: **Mahmoud Muhammad**
-- Github: [@mahmoud717](https://github.com/mahmoud717)
-- Twitter: [@mahmoud](https://twitter.com/mahmoud26369406)
-
-:bust_in_silhouette:**Gürbüz Ertunga**
-- Github: [@gurbuzertunga](https://github.com/gurbuzertunga)
-- LinkedIn: [@Gurbuz Ertunga](https://www.linkedin.com/in/gurbuz-ertunga-a607a2a5/)
-## :handshake: Contributing
-
-## Show your support
-If you've read this far, you must like the project! Give us a :star:️!
-## Acknowledgments
-- This Project was part of an assignment available on The Odin Project.
-- Our thanks to Microverse and all our peers and colleagues there.
-## :memo: License
+# The Tic-Tac-Toe
+
+> In this project, we are building a Tic Tac Toe game. The main goal is to put into practice the main concepts of Object Oriented Programming. Particularly, classes, modules and objects and access their attributes.
+
+For this project, we wrote a Ruby executable script main.rb that allows the user to play the game of Tic, Tac, Toe in the Terminal.
+
+## How To Play
+
+'Tic, Tac, Toe' is a traditional game for two players played on a board with 9 squares.
+
+Each player is represented by a symbol, either 'X' or 'O'. The players then take turns to place their respective symbols in an empty square on the board, with O's going first - in our version of the game, these squares are represented by the numbers 1 to 9.
+
+The aim of the game is to have three of your symbols form a line, be it vertically, horizontally or diagonally, as illustrated in the image below.
+
+![A Winning Combination of Tic, Tac, Toe](.github/winning-screenshot.webp)
+
+In the event that all 9 squares are taken without either player getting three of their symbols in a line, a draw is declared.
+
+### Accessing The Game
+
+- To quickly play the game, follow the link in the [Live Demo section](https://repl.it/@gurbuzertunga/tic-tac-toe#bin/main.rb) and hit the Run button in the header. The game will begin in the window on the right.
+
+- To download the contents of this repository and play the game locally, follow the steps in the ['Local Installation' section](#local-installation).
+
+## Built With
+
+- Ruby
+- VSCode
+- Rubocop
+
+## Live Demo
+
+To quickly see our project in action, visit the [live demo]('#') on repl.it.
+
+There are no pre-requisites for enjoying the game through this method.
+
+## Local Installation
+
+### Prerequisites
+
+If you intend to download the project, you will need to have Ruby already installed on your machine. For more information on how to install Ruby, follow [this link](https://www.ruby-lang.org/en/downloads/).
+
+### Installation Instructions
+
+To get your own copy of our project simply clone the repository to your local machine.
+
+**Step 1**: Using the Command Line, navigate into the location where you would like to have the repository. Then enter the following line of code:
+
+`git clone https://github.com/gurbuzertunga/tic-tac-toe.git`
```

Comment:
- [ ] **OPTIONAL** You can also use 3 backquotes at the beginning and 3 backquotes at the end to format it this way

```
This is an example
```
![image](https://user-images.githubusercontent.com/53256139/91446771-7f9f0b00-e83d-11ea-855e-635ad080732e.png)